lib/server.js

```diff
@@ -48,23 +78,36 @@ server.init = function(options) {
 server.start = function() {
 	util.log('Starting Prerender');
 	this.isBrowserConnected = false;
-	this.startPrerender().then(() => {
 
-		process.on('SIGINT', () => {
-			this.killBrowser();
-			setTimeout(() => {
-				util.log('Stopping Prerender');
-				process.exit();
-			}, 500);
+	return new Promise((resolve, reject) => {
+		this.appServerReference = this.app.listen(this.expressOptions, () => {
```

Comment:
I do like this better for being able to start/stop for tests, but since the `lib/index.js` used to call app.listen and then `server.start()` would actually start the Chrome server, does that make this a breaking change requiring a major version bump? I'd argue no one would actually include `lib/index.js` without calling `server.start()` since that would have been useless, but just a thought.

Reply:
That's a very good point, thanks! Actually we can still keep it backwards compatible, if i'm not mistaken. The ```lib/index.js``` is not an entry point of the app, so it may be good as is now. However the ```./index.js``` is a de-facto entry point, where I added the ```server.start()``` invocation now. And the ```server.js``` still does what we need, if I didn't mess it up :)

What do you think?